web/src/pages/billing/BillingCostByMonth.tsx

```diff
@@ -426,6 +484,7 @@ const BillingCostByTime: React.FunctionComponent = () => {
                                 minWidth: '115px',
                                 height: '36px',
                             }}
+                            buttonText="Topics"
```

Comment:
Can we remove the column visibility select as it is kind of doublling up with the topic selection